tools/scripts/CaptureDevHomeLogs.ps1

```diff
@@ -0,0 +1,18 @@
+# Quick script to capture log files and create a zip file for upload to GitHub
+# Currently this only creates one based on Date. We could add a time, so that 
+# the ISV can have more than one Zip file
+
+$MyZip=$(Get-Date -Format yyyy-MM-dd) + "DevHome.Zip"
+$LogPath="C:\Users\" + $Env:UserName  + "\AppData\Local\Packages"
+$LogFiles = Get-ChildItem -Path $LogPath -Filter "*.dhlog" -recurse

```

Comment:
This can grab multiple log files in the same package.  Consider adding a filter to get the most recent log file.

Reply:
Recommend grabbing all the files. Older files will have context that may be important to identifying a problem. If this gets too many, could add a script parameter to add the latest or all, but I don't think that's necessary for initial checkin.